### PR TITLE
Preview network re-spin

### PR DESCRIPTION
**Pull request by pgrange**
Fetched configuration for the new preview network from https://book.world.dev.cardano.org/environments.html#preview-testnet

When trying to reset my node to synchronize with the new preview network I got the following error:
```
Wrong Byron genesis file: the actual hash is "83de1d7302569ad56cf9139a41e2e11346d4cb4a31c00142557b6ab3fa550761", but the expected Byron genesis hash given in the node configuration file is "72593f260b66f26bef4fc50b38a8f24d3d3633ad2e854eaf73039eb9402706f1"
```

After fetching preview configuration again from [Cardano book](https://book.world.dev.cardano.org/environments.html#preview-testnet), the problem is solved.

### CODE REVIEW

**Comment by ch1bo** [comment]
@rhyslbw @KtorZ or whoever is maintainer on this repo. Please invoke the https://github.com/input-output-hk/cardano-configurations/actions/workflows/refresh-configurations.yaml workflow or merge this PR.

Alternatively, give more people write permissions to this repository :)

File: network/preview/cardano-node/config.json [hunk 1]
```diff
@@ -4,22 +4,23 @@
   "ApplicationName": "cardano-sl",
   "ApplicationVersion": 0,
   "ByronGenesisFile": "../genesis/byron.json",
-  "ByronGenesisHash": "72593f260b66f26bef4fc50b38a8f24d3d3633ad2e854eaf73039eb9402706f1",
-  "EnableP2P": false,
+  "ByronGenesisHash": "83de1d7302569ad56cf9139a41e2e11346d4cb4a31c00142557b6ab3fa550761",
+  "EnableP2P": true,
```
Comment (ch1bo) [inline network/preview/cardano-node/config.json:8]: FYI: `preview` is the non-p2p version of the config and we have a `preview_p2p` variant in this repo. You copied the p2p version here (which might be the only working right now?)

**Comment by KtorZ** [comment]
I've refreshed the configurations manually: https://github.com/input-output-hk/cardano-configurations/actions/runs/3377031799 

The workflows runs once a day, so it'd have been automatically "fixed" by tomorrow.

File: network/preview/cardano-node/config.json (hunk 1 above)
Comment (pgrange) [reply network/preview/cardano-node/config.json:8]: I copy/pasted what is available from [Cardano book](https://book.world.dev.cardano.org/environments.html#preview-testnet) but I have pretty much no idea of what I am doing here, except it works 🤷 
Feel free to improve or reject this P.R.

**Pull request closed without merging**